native/packages/react-native-bpk-component-phone-input/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import BpkDialingCodeList from './src/BpkDialingCodeList';
+import BpkDialingCodeListItem from './src/BpkDialingCodeListItem';
+
```

Comment:
we should expose the PropsTypes here see https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-button/index.js#L23

Reply:
The flow types you mean?

Reply:
Have now done so 👍